packages/operator/docs/docs.go

```diff
@@ -70,7 +70,6 @@ var doc = `{
                         "in": "body",
                         "required": true,
                         "schema": {
-                            "type": "object",
```

Comment:
What version of swagger do you use? 
https://github.com/odahu/odahu-flow/blob/develop/.travis.yml#L76

Reply:
(venv) EPUAKHAW1608:operator vlad$ swag --version
swag version v1.6.3